### PR TITLE
Adjacent elements Product in python

### PR DESCRIPTION
**Pull request by shloka-gupta**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Description

(When given an array of integers, we have to find the pair of adjacent elements that have the largest product and return that product.)

### Checklist

- [Yes] I've read the contribution guidelines.
- [Yes] I've checked the issue list before deciding what to submit.
- [Yes] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Issue #2236)

### CODE REVIEW

**Comment by shloka-gupta** [comment]
Could you please review this PR?
@atarax665 

(And could you add the GSSOC21 label?)

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
- Add Time and Space Complexity at the end of the Code   
- Add new line at the EOF

File: Python/cp/adjacent_elements_product.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Enter the numbers : -5 -3 -2
+Max=  15  product of elements at position  0 , 1
+
+Time Complexity : O(n)
+Space Complexity : O(1)
+"""
+
+
+
+
```
Comment (ankitaggarwal23) [inline Python/cp/adjacent_elements_product.py]: Remove this extra lines one line is enough

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Rest looks good to merge

**Comment by ankitaggarwal23** [comment]
Resolve conflicts

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**